### PR TITLE
Update pubspecs for dart2.0 and angular 5 stable

### PR DESCRIPTION
**Pull request by alorenzen**
Note, I still need to update the CHANGELOGs. But running this through travis while I work on those.

### CODE REVIEW

**Comment by kevmoo** [comment]
Leave the SDK lower-bound constraints as-is. There may be folks unable to roll to stable due to dependency issues.

File: .travis.yml [hunk 1]
```diff
@@ -48,6 +48,7 @@ jobs:
   # See https://github.com/dart-lang/angular/issues/1114.
   allow_failures:
     - env: PKG="examples/github_issues"
+    - env: PKG="examples/hacker_news_pwa"
```
Comment (kevmoo) [inline .travis.yml]: This doesn't seem to affect anything...

Comment (alorenzen) [reply .travis.yml]: Yes it does. It allows travis to continue to follow-up stages even if this one job had failures.

Comment (alorenzen) [reply .travis.yml]: `Your build matrix was set to allow the failure of job 5044.14 so we continued this build to the next stage.`

**Comment by kevmoo** [comment]
I was used to something showing up on the display to show 'allow_failures'
– just not w/ build steps?

**Pull request closed without merging**